components/layout/__tests__/navigation.test.tsx

```diff
@@ -5,143 +5,31 @@ import { bottomNavigation } from "~/lib/layout/navigation-data";
 
 jest.mock("next/router", () => require("next-router-mock"));
 
+const navigations = bottomNavigation.map((item) => [
+  item.name,
+  item.href,
+  Boolean(item.external),
+]);
+
 describe("Navigation", () => {
-  it("renders the styling correctly", () => {
-    const { container } = render(<Navigation />);
+  it.each(navigations)(
+    "should render %s and link it to %s correctly",
+    (name, href, external) => {
+      render(<Navigation />);
+      const navigationItem = screen.getByText(name as string);
+      const navigationLink = navigationItem.closest("a");
```

Comment:
<img width="1506" alt="Screenshot 2021-08-15 at 23 39 07" src="https://user-images.githubusercontent.com/6315466/129484071-4edc9f13-e933-4a4b-80a1-e52e341bfd5a.png">

Could you please use the query recommended by the testing-playground.com Chrome extension here?

```suggestion
      const navigationLink = screen.getByRole('link', { name: item.name })
```